src/components/molecules/modals/ShopModal.js

```diff
@@ -324,6 +314,9 @@ export const ShopModal = ({
     setValidEmail(true)
   }
 
+  const handleEnableButton = () => setButtonDisabled(false)
```

Comment:
Couldn't use only const handleButtonStatus and change the state accordingly?

Reply:
seemed to me more readable this way